Documentation/ApiOverview/Backend/BackendRouting.rst

```diff
@@ -119,6 +120,49 @@ The request sequence in the TYPO3 Core looks like this:
    to be aware of mentioned implications when implementing custom web applications.
 
 
+..  index:: Backend routing; Dynamic URL parts
+..  _backend-routing-dynamic-parts:
+
+Dynamic URL parts in backend URLs
+=================================
+
+..  versionadded:: 12.1
+
+Backend routes can be registered with path segments that contain dynamic parts,
+which are then resolved into a PSR-7 request attribute called
+:ref:`routing <typo3-request-attribute-routing-backend>`.
+
+These routes are defined within the route path as named placeholders:
+
+..  code-block:: php
+    :caption: EXT:my_extension/Configuration/Backend/Routes.php
+
+    use MyVendor\MyExtension\Controller\MyRouteController;
+
+    return [
+        'my_route' => [
+            'path' => '/my-route/{identifier}',
+            'target' => MyRouteController::class . '::handle',
+        ],
+    ];
+
+Within the controller:
+
+..  code-block:: php
+    :caption: EXT:my_extension/Classes/Controller/MyRouteController.php
```

Comment:
What kind of controller? Exbase controller? Just any? Does it have to extend something? I would show the complete class including inheritance

Reply:
That does not matter. Extbase or not, it is about the request object attribute.

Reply:
I know that. But newbies get easily confused: Sometimes when we talk about controllers we are talking about Extbase controller, sometimes just about a Controller in the MVC sense, we should clarify this when we talk about a controller

Reply:
Added information.